packages/atomic/src/pages/v1.html

```diff
@@ -386,6 +386,7 @@
           </atomic-result-template>
         </atomic-result-list-v1>
         <div class="footer-item side-by-side-container">
+          <atomic-load-more-results></atomic-load-more-results>
           <atomic-pager></atomic-pager>
           <atomic-results-per-page></atomic-results-per-page>
```

Comment:
Not to sure how to include this component since the pager is already there.

Reply:
We will probably showcase `atomic-load-more-results` more so we can remove the other pagination components here.